meta-mender-tegra/meta-mender-tegra-jetpack5/recipes-bsp/tegra-binaries/tegra-bootfiles_%.bbappend

```diff
@@ -1,5 +1,29 @@
-DEPENDS += "mender-custom-flash-layout"
-MENDER_PARTITION_FILE = "${STAGING_DATADIR}/mender-flash-layout/flash_mender.xml"
-PARTITION_FILE = "${MENDER_PARTITION_FILE}"
 
+do_install:append() {
+    cat <<EOF >${WORKDIR}/UDA.xml
+<partition_layout>
+    <device>
+        <partition name="UDA">
+            <filename> DATAFILE </filename>
+        </partition>
+    </device>
+</partition_layout>
+EOF
```

Comment:
This could be useful for other projects which want/need a data partition and might be good to upstream into meta-tegra at some point.  It could be nice to just support this directly in nvflashxmlparse too with `--add-data-partition` or similar.

@madisongh FYI

Reply:
That would be nice @dwalkes